cmd/bnsd/x/blueaccount/msg.go

```diff
@@ -155,6 +156,23 @@ func (msg *DeleteAccountMsg) Validate() error {
 	return errs
 }
 
+var _ weave.Msg = (*FlushDomainMsg)(nil)
+
+func (FlushDomainMsg) Path() string {
+	return "blueaccount/delete_all_accounts_msg"
+}
+
+func (msg *FlushDomainMsg) Validate() error {
+	var errs error
+	errs = errors.AppendField(errs, "Metadata", msg.Metadata.Validate())
+	// Domain validation rules are dynamically set via configuration and
+	// cannot be enforced here.
+	if len(msg.Domain) == 0 {
```

Comment:
Don't we have a general regex for domain string validations?

Reply:
Yes. It's declared in configuration, which is not available to validation method.